configs/lwip/lwipopts.h

```diff
@@ -498,7 +502,7 @@ Certain platform allows computing and verifying the IP, UDP, TCP and ICMP checks
 /* Added by Realtek end */
 
 /* Extra options for lwip_v2.0.2 which should not affect lwip_v1.4.1 */
-#define LWIP_TCPIP_CORE_LOCKING         0
+#define LWIP_TCPIP_CORE_LOCKING         1
```

Comment:
And we don't need to update this value because it's updated in [line#563](https://github.com/awslabs/freertos-webrtc-reference-on-amebapro-for-amazon-kinesis-video-streams/blob/60aec9fab624cc0dae5f2b9cccd789a81cc5c029/configs/lwip/lwipopts.h#L563) with ENABLE_AMAZON_COMMON defined.